slices/gawk.yaml

```diff
@@ -0,0 +1,19 @@
+package: gawk
+
```

Comment:
What do you think about the `/usr/include/gawkapi.h` file? Should it be in any slices?

Reply:
Are header files in scope for chiseled containers? I didn't see anything else that was including them (at least in `22.04`) and couldn't think of a good use case for someone doing builds inside a chiseled container.

I could add a `headers` (or maybe `dev`?) slice if there's something I'm missing though.

Reply:
Added a `headers` slice, let me know what you think.

Reply:
Good point. I was just curious if we should start including them, but for now, there seems not to be much application for that. 
I will let you decide if you want to remove the `headers` slice or not. It's good for me either way.